domain_models/fields.py

```diff
@@ -55,6 +55,7 @@ def get_value(self, model, default=None):
 
         :param DomainModel model:
         :param mixed default:
+        :return mixed:
```

Comment:
There is no mixed type in Python. In current case you can use `object` instead.

Also `:return:` should contain a description of returning value, while `:rtype:` contains type only